vueapp/src/views/TilePad.vue

```diff
@@ -76,12 +82,18 @@ export default {
 		Tile,
 		Sentence,
 		draggable,
-		EditModeHeader
+		EditModeHeader,
+		vueCustomScrollbar
 	},
 	data () {
 		return {
 			tileData: TileData,
 			sentenceTiles: [],
+			settings: {
```

Comment:
Similar to my previous question on `settings`, can we remove this?

Reply:
I believe we can.